### PR TITLE
feat!: remove Transifex calls for FC-0012

### PR DESCRIPTION
**Pull request by OmarIthawi**
## Breaking change!

This change breaks the Jenkins transifex integration which has been deprecated in favor of the new GitHub Transifex App integration as part of [OEP-58](https://open-edx-proposals.readthedocs.io/en/latest/architectural-decisions/oep-0058-arch-translations-management.html#specification).

## Changes

 - [x] Removes direct use of `tx pull` and `tx push` commands from the [XBlock to let Open edX manage it](https://github.com/openedx/edx-platform/pull/33166).
 - [x] Remove source and language translations from the repositories, hence no `.po` or `.mo` files will be committed into the repos. 
 - [x] Remove Transifex related `Makefile` targets and configuration files.
 - [x] Use the OEP-58 JavaScript translations from the Open edX platform

Merge timeline
-----------------------

This should only be merged after [Translation Infrastructure update OEP-58](https://open-edx-proposals.readthedocs.io/en/latest/architectural-decisions/oep-0058-arch-translations-management.html#specification) is fully implemented.

The timing announcement will be shared by @brian-smith-tcril on [#translations-project-fc-0012](https://openedx.slack.com/archives/C04R6TUJB7T) Open edX Slack channel.

Keep this pull request as a draft to prevent accidental merge.

### Pre-merge checklist

 - [ ] https://github.com/openedx/wg-translations/issues/20 is approved

References
----------

This contribution is part of the [FC-0012 project](https://openedx.atlassian.net/l/cp/XGS0iCcQ) which is sparked by the [Translation Infrastructure update OEP-58](https://open-edx-proposals.readthedocs.io/en/latest/architectural-decisions/oep-0058-arch-translations-management.html#specification).

Up-to-date project overview and details are available in the [Approach Memo and Technical Discovery: Translations Infrastructure Implementation](https://docs.google.com/document/d/11dFBCnbdHiCEdZp3pZeHdeH8m7Glla-XbIin7cnIOzU/edit#) document.

Join the conversation on [Open edX Slack #translations-project-fc-0012](https://openedx.slack.com/archives/C04R6TUJB7T).

Check the links above for full information about the overall project.

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @OmarIthawi!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


This is currently a draft pull request. When it is ready for our review and all tests are green, click "Ready for Review", or remove "WIP" from the title, as appropriate.

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @OmarIthawi!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


This is currently a draft pull request. When it is ready for our review and all tests are green, click "Ready for Review", or remove "WIP" from the title, as appropriate.

**Comment by itsjeyd** [comment]
@OmarIthawi Thank you for this contribution! Let us knnow when the changes are ready for review.

**Comment by OmarIthawi** [comment]
@itsjeyd, thanks!

This pull request will wait until deprecation notice has been communicated properly. There will be a total of 40 other similar pull request grouped into one issue:

 - https://github.com/openedx/wg-translations/issues/20
 
TL;DR; we'll let you know once it's ready.

**Comment by itsjeyd** [comment]
@OmarIthawi Oh wow, 40 PRs! 😅 Thanks for the update, that's good to know.

**Comment by itsjeyd** [comment]
Hey @OmarIthawi, it's been a while so I wanted to check in and see if you had any updates about this PR (and the batch of related ones that you mentioned [above](https://github.com/openedx/xblock-drag-and-drop-v2/pull/352#issuecomment-1766899435))?

**Comment by OmarIthawi** [comment]
Not yet @itsjeyd. This deprecation process will likely to be done after Redwood. We don't have a specific timeline for merging those pull requests given the mandatory upfront announcements we'd need to send the community and 2U:

 - https://github.com/openedx/wg-translations/issues/20

**Comment by OmarIthawi** [comment]
@itsjeyd this pull request is finally ready, and only needs coordination with the community.

It's likely to be merged after Redwood with all other pull requests listed in:

 - https://github.com/openedx/wg-translations/issues/20

`@brian-smith-tcril` will be leading the merge and deprecation process.

**Comment by itsjeyd** [comment]
Sounds good @OmarIthawi, thanks for the update!

**Comment by itsjeyd** [comment]
@mphilbrick211 I went ahead and added `waiting for Redwood` to last PM check notes for all the PRs listed in https://github.com/openedx/wg-translations/issues/20, as well as https://github.com/openedx/frontend-app-support-tools/pull/361.

CC @OmarIthawi

**Comment by mphilbrick211** [comment]
Thank you, @itsjeyd!

**Comment by OmarIthawi** [comment]
@brian-smith-tcril @timmc-edx @itsjeyd it's finally ready for merge :)

**Review by brian-smith-tcril — APPROVED** [review]

**Comment by openedx-webhooks** [comment]
@OmarIthawi 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/xblock-drag-and-drop-v2&entry.867055334=https://github.com/openedx/xblock-drag-and-drop-v2/pull/352&entry.1484655318=https://github.com/OmarIthawi&entry.752974735=2023-09-26+08:09&entry.1917517419=2024-03-18+19:05&entry.2133058324=Yes) so we can improve your experience in the future.

**Comment by openedx-webhooks** [comment]
@OmarIthawi 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/xblock-drag-and-drop-v2&entry.867055334=https://github.com/openedx/xblock-drag-and-drop-v2/pull/352&entry.1484655318=https://github.com/OmarIthawi&entry.752974735=2023-09-26+08:09&entry.1917517419=2024-03-18+19:05&entry.2133058324=Yes) so we can improve your experience in the future.
